packages/devtools-launchpad/src/components/LandingPage.js

```diff
@@ -154,6 +152,4 @@ const LandingPage = React.createClass({
   }
 });
 
-module.exports = connect(
-  state => ({ tabs: getTabs(state) })
-)(LandingPage);
+module.exports = LandingPage;
```

Comment:
You should be able to keep this here and reference the wrappedCompondnt or something like that